hal/hal.spec

```diff
@@ -6,10 +6,10 @@
 @ fastcall -arch=i386 ExTryToAcquireFastMutex(ptr) ntoskrnl.ExiTryToAcquireFastMutex
 @ stdcall HalAcquireDisplayOwnership(ptr)
 @ stdcall HalAdjustResourceList(ptr)
+@ stdcall HalAllProcessorsStarted()
 @ stdcall HalAllocateAdapterChannel(ptr ptr long ptr)
 @ stdcall HalAllocateCommonBuffer(ptr long ptr long)
 @ stdcall HalAllocateCrashDumpRegisters(ptr ptr)
-@ stdcall HalAllProcessorsStarted()
```

Comment:
Why moving it up? (it's not in lowercase-lexicographical order anymore)

Reply:
No, it's in the correct order now, where uppercase comes before lowercase.